pkg/models/postgres/schema_migration.sql

```diff
@@ -0,0 +1,17 @@
+--liquibase formatted sql
```

Comment:
Move to a `SQL` directory in the root of the project. Create an accompanying readme to link to the `models` and `postgres` readme files.